packages/core-mobile/app/vmModule/ModuleManager.ts

```diff
@@ -174,7 +174,12 @@ class ModuleManager {
         ? BlockchainId._2JVSBOINJ9C2J33VNTVZ_YT_VJNZD_N2NKIWW_KJCUM_HUWEB5DB_BRM
         : BlockchainId._2O_YMBNV4E_NHYQK2FJJ_V5N_VQLDBTM_NJZQ5S3QS3LO6FTN_C6FBY_M
     }
-    return BlockchainId._11111111111111111111111111111111LPO_YY
+    if (vmName === NetworkVMType.PVM)
+      return BlockchainId._11111111111111111111111111111111LPO_YY
+
+    return isTestnet
+      ? BlockchainId.Y_H8D7TH_NJKXMTKUV2JG_BA4P1RN3QPR4P_PR7QYNFCDO_S6K6HWP
+      : BlockchainId._2Q9E4R6MU3U68N_U1F_YJGB_R6JVWR_RX36COHP_AX5UQXSE55X1Q5
```

Comment:
can you add a comment for what these 2 are for? I assume for C?

Reply:
yes. they are for c chains. added comments.